sympy/core/sympify.py

```diff
@@ -273,6 +272,31 @@ def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
     >>> kernS(s)
     -2*(-(-x + 1/x)/(x*(x - 1/x)**2) - 1/(x*(x - 1/x))) - 1
 
+    Parameters
+    ==========
+
+    a : Can be :
+        - any object defined in SymPy
+        - standard numeric python types: int, long, float, Decimal
+        - strings (like "0.09" or "2e-19")
+        - booleans, including ``None`` (will leave ``None`` unchanged)
+        - dict, lists, sets or tuples containing any of the above
+
+    convert_xor : boolean, optional
+        If true, treats XOR, $^$, as exponentiation, $**$. Used when input is a string
```

Comment:
what happens when this is False?

Reply:
It does not treat ^ as exponetiation. Should I mention that?

Reply:
Done.